lib/MusicBrainz/Server/Data/WebService.pm

```diff
@@ -202,16 +202,16 @@ sub xml_search
         };
     }
 
-    my $format = ($args->{fmt} // "") eq "json" ? "jsonnew" : "xml";
-
-    my $url_ext = "/ws/2/$resource/?" .
-        "max=$limit&type=$resource&fmt=$format&offset=$offset" .
-        "&query=" . uri_escape_utf8($query) . "&dismax=$dismax";
+    my $format = ($args->{fmt} // "") eq "json" ? "mbjson" : "mbxml";
+    my $defType = $dismax ? 'lucene' : 'dismax';
```

Comment:
`$defType` is set to `'dismax'` if `$dismax` is false?

Reply:
fixed.

Reply:
fixed.